### ISSUE TITLE
Unicode Ellipsis (U+2026) is wrongly shown as dot in the editor

### ISSUE THREAD

**Issue by FreeHarry** [microsoft/vscode#148807]
Issue Type: <b>Bug</b>

Unicode Ellipsis (U+2026) is wrongly shown as dot in the editor. This is the case only for the editor. The same character is displayed correctly e.g. in the find widget. See screenshot.

<img width="783" alt="2022-05-05_16-46-11--Code" src="https://user-images.githubusercontent.com/24735031/166951339-7ae32233-5113-45bb-82af-f60e354b570c.png">

VS Code version: Code 1.66.2 (dfd34e8260c270da74b5c2d86d61aee4b6d56977, 2022-04-11T07:46:01.075Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.82GB (18.02GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (42)</summary>

Extension|Author (truncated)|Version
---|---|---
calculate|aca|2.1.0
Bookmarks|ale|13.2.4
openmatchingfiles|bca|0.5.2
insert-unicode|bru|0.13.3
bracket-select|chu|2.0.2
python-snippets|cst|0.1.2
xml|Dot|2.5.1
vscode-open-in-browsers|fab|1.1.7
seito-openfile|Fr4|1.8.8
list-symbols|jmb|0.0.5
vscode-insertdatestring|jsy|2.3.1
rainbow-csv|mec|2.3.0
ecdc|mit|1.7.0
vscode-filesize|mkx|3.1.0
python|ms-|2022.4.1
vscode-pylance|ms-|2022.4.3
jupyter|ms-|2022.3.1000901801
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.6
cpptools|ms-|1.9.8
hexeditor|ms-|1.9.6
autodocstring|njp|0.6.1
favorites|ole|1.5.15
vscode-print|pdc|0.9.14
linesinfostatusbar|pet|0.0.7
material-icon-theme|PKi|4.16.0
calc|rai|2.1.0
color-manager|roy|0.6.3
text-marker|ryu|1.11.0
vscode-active-file-status|sgt|2.0.0
vscode-table-formatter|shu|1.2.1
vscode-fileutils|sle|3.5.0

vscode-autohotkey|sle|0.2.2
numbermonger|sof|0.5.0
rewrap|stk|17.8.0
code-spell-checker|str|2.1.11
code-spell-checker-german|str|2.0.3
vscode-markdown-paste-image|tel|0.19.0
multi-cursor-search|yo1|2.1.0
markdown-all-in-one|yzh|3.4.3
cursorCharCode|zei|0.2.0
material-theme|zhu|3.13.24

**Comment by IllusionMH** [comment]
/needsMoreInfo

What font are you using?

Unable to reproduce it with Cascadia Code PL( on the screenshot) and Consolas (default)
![image](https://user-images.githubusercontent.com/1312662/166964922-1fbbf8ca-aab1-4bbe-ab3d-1e715fb8bc50.png)

**Comment by FreeHarry** [comment]
Embarrassing. Yes, that is what I forget to check.

I use "Noto Sans Mono" font. The weird thing was, that on my windows machine I had that bug, but on my linux machine not. So I figured out that the Noto-font was updated in the meantime and I was using an old one which had this bug (see [here](https://github.com/googlefonts/noto-fonts/issues/1035)). After updating the font family on my windows machine, now the ellipsis character is displayed correctly.

Sorry for the inconvenience.
